freedomofpress/securedrop-workstation
# Update to signing key with 2023 expiry

## Status

Ready for review

## Test plan

General verification:
- [ ] Confirm that the key in this PR corresponds to the updated SecureDrop release key with 2023 expiry. https://securedrop.org/securedrop-release-key.asc

On Qubes 4.0:
- If you have an existing install:
  - Run the command `gpg /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation`.
  -  [ ]  Confirm that you're either seeing the test key, or a key with the 2022 expiry.
  - For bonus points, double-check that the key isn't hiding in RPM's keyring with the command `rpm -q gpg-pubkey --qf '%{NAME}-%{VERSION}-%{RELEASE}\t%{SUMMARY}\n'`.
- `make clone` from this branch to build an RPM
- Install this RPM manually using `sudo dnf install` (it will be generated in `rpm-build/RPMS/noarch` after `make clone`).
- Configure your environment in `/usr/share/securedrop-workstation-dom0-config/`. Ensure the environment `config.json` is set to `prod`
- Run `sdw-admin --apply`.
- Repeat the `gpg` verification from earlier.
- [ ] Confirm that you're now seeing the updated key.
- [ ] Confirm that `make test` still runs successfully or that no new tests are failing.

On Qubes 4.1:
- This won't be testable until #784 lands. After that, the test procedure should be identical.
